src/app/page/Home/Home.js

```diff
@@ -80,62 +144,100 @@ const HomePage = ({ auth, history }) => {
+            <img src={HomeImage5} alt="" />
+          </Box>
+          <Box className={classes.WantToHelpSection} px={8} py={4}>
+            <Box className={classes.WhiteBgText}>
+              <Heading>Want to help?</Heading>
+            </Box>
+          </Box>
+          <Box p={2} textAlign="left">
+            <Text>
+              Sign up to join your local network helping neighbors through this crisis. Deliver food, medicine, and supplies to the most vulnerable.
+            </Text>
+            <Box textAlign="center" pt={2}>
+              <Button
+                onClick={() => history.push("/signup")}
+                data-testid="btn-signup"
+                className={classes.SignupButton}
+              >
+                Volunteer
+            </Button>
+            </Box>
+          </Box>
+          <Box className={classes.PurpleBg} textAlign="left" p={2}>
+            <Box>
+              <Heading color="white" style={{ display: "inline", verticalAlign: "middle" }}>Donate</Heading>
+              <img src={Emoticon} className={classes.Emoticon} alt="smiley"></img>
+            </Box>
+            <Box my={1}>
+              <Text color="white">Many families can't afford fresh food. Please help sponsor food boxes for those in need.</Text>
+            </Box>
+            <Box textAlign="center">
+              <Button
+                onClick={() => history.push("/signup")}
+                data-testid="btn-donate-funds"
+                className={classes.SignupButton}
+                color="secondary"
+              >
+                Donate Funds
             </Button>
-          </Grid>
-          <Card>
-            <Grid container>
-              <H2 className={classes.Paragraph}>How it works</H2>
-              <Grid container>
-                <Body1>Need help?</Body1>
-                <Body1 className={classes.Paragraph}>
-                  Sign up to make a request for a food box, small errand, or a pharmacy pickup. Your
-                  ask goes right to the local coordinator, and is matched with a volunteer who will
-                  take care of your need ASAP.
-                </Body1>
-                <Body1>Want to help?</Body1>
-                <Body1 className={classes.Paragraph}>
-                  Sign up to join your local network of volunteers helping their neighbors through
-                  this crisis. Deliver food, medicine, and other supplies to the most vulnerable.
-                </Body1>
-              </Grid>
-              <Grid container>
-                <img src={HomeImage2} className={classes.HomeImage} alt="" />
-              </Grid>
-            </Grid>
-          </Card>
-        </Grid>
+            </Box>
+          </Box>
+          <Box className={classes.RequestByPhoneBg}>
+            <Card className={classes.PhoneCard}>
+              <img src={PhoneIcon} alt="call"></img>
+              <Text>To request food by phone, call</Text>&nbsp;<PhoneNo>555-555-5555.</PhoneNo>
```

Comment:
Can we use an anchor tag here for the number instead?
you can add `tel:` to href to make auto call that number
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a